installing/installing_aws/ipi/installing-aws-secret-region.adoc

```diff
@@ -13,6 +13,11 @@ In {product-title} version {product-version}, you can install a cluster on Amazo
 
 To configure a cluster in either region, you change parameters in the `install config.yaml` file before you install the cluster.
 
+[WARNING]
+====
+In {product-title} {product-version}, the installation program uses Cluster API (CAPI) instead of Terraform to provision cluster infrastructure during installations on AWS. Installing a cluster on AWS into a secret or top secret region has not been tested with CAPI as of the release of {product-title} {product-version}. This document will be updated once installation into a secret region has been tested.
+====
```

Comment:
```suggestion
In {product-title} {product-version}, the installation program uses Cluster API (CAPI) instead of Terraform to provision cluster infrastructure during installations on AWS. Installing a cluster on AWS into a secret or top-secret region has not been tested with CAPI as of the release of {product-title} {product-version}. This document will be updated after installation into a secret region has been tested.
====
```
As per the [IBM guidelines](https://www.ibm.com/docs/en/ibm-style?topic=word-usage), use the word 'once' only to mean one time. Please don't use it as a conjunction to mean after or when.